graph-db/src/bootstrapGremlinServer/handler.js

```diff
@@ -0,0 +1,9 @@
+const { bootstrapGremilinServer } = require('../utils/bootstrapGremlinServer');
+
+module.exports.bootstrap = async event => {
+  let body = await bootstrapGremilinServer();
```

Comment:
I'm assuming `body` will be used eventually? Otherwise you can just `await bootstrapGremlinServer();` without the assignment